### PR TITLE
Wj branch

### PR DESCRIPTION
**Pull request by wjlim-14**
add : add date and modify email recipients

### CODE REVIEW

File: pipeline/wj_luigi.py [hunk 1]
```diff
@@ -128,12 +129,12 @@ def run(self):
...
         for i in range(5):
             message += '|'.join(map(str, df.values.tolist()[:5][i])) + '\n'
-        '''
 
         msg = EmailMessage()
+        msg["Date"] = datetime.now() -+ timedelta(hours=8)
         msg['Subject'] = subject
         msg['From'] = email().username
-        msg['To'] = ', '.join(receipients)
+        msg['To'] = receivers().email
```
Comment (geraldooi) [inline pipeline/wj_luigi.py:137]: I didn't think of this way. btw, nice idea.

File: pipeline/wj_luigi.py [hunk 2]
```diff
@@ -128,12 +129,12 @@ def run(self):
         ''')
         for i in range(5):
             message += '|'.join(map(str, df.values.tolist()[:5][i])) + '\n'
-        '''
 
         msg = EmailMessage()
+        msg["Date"] = datetime.now() -+ timedelta(hours=8)
```
Comment (geraldooi) [inline pipeline/wj_luigi.py:134]: Can you explain to me why see jie and you both using the the same -+?

Comment (wjlim-14) [reply pipeline/wj_luigi.py:134]: Ah, the code is wrong, it suppose to be (datetime.now() - timedelta(hours=8)).
We got discuss together yesterday and there is a mistake that I delete the wrong row while testing.

Comment (geraldooi) [reply pipeline/wj_luigi.py]: I see. OK.

**Pull request merged**